plugins/@grouparoo/pipedrive/src/migrations/000001-plugin-pipedrive-syncMode.ts

```diff
@@ -0,0 +1,14 @@
+export default {
+  up: async function (migration) {
+    await migration.sequelize.transaction(async () => {
+      // Set all existing instances to "sync" to maintain same behavior
+      await migration.sequelize.query(
+        `UPDATE "destinations" SET "syncMode"='sync' WHERE "state"='ready' AND "type"='pipedrive-export' AND "locked" IS NULL`
+      );
```

Comment:
Note to our future selves:
* We didn't modify those Apps configured via code-config, because as the application restarts immediately after this migration Grouparoo will then try to apply the code-config, and an error will be thrown about a missing `syncMode`.
* We don't want to modify people's code-config files - the error about a (now) required `syncMode` is the best we can do. 

To discuss: 
* Is this a breaking change (ie: starts the `v0.3.x` line of Grouparoo?